src/main/perl/RuleBasedEditor.pm

```diff
@@ -223,7 +231,7 @@ use enum qw(
 
 =pod
 
-=head3 LINE_VALUE_OPT_xxx: options for rendering the value
+=head3 LINE__VALUE_OPT_xxx: options for rendering the config line
```

Comment:
It has been fixed during the rename of these options, It seems your comment doesn't apply to last version in the PR...